### DISCUSSION TITLE
minimum order quantity

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#236] [General]
Hi
Is it possible to set up a product that can be ordered in any amount, the only restriction is that there is a minimum order of 10.
Example, a school could order 13 or 64 books but not 4.

Regards
Roger

**Comment by DNNMonster** [comment]
It's not currently implemented but I think it's a good feature.  I pushed a branch up that we'll let other eyeballs review before merging it into develop.
